package.json

```diff
@@ -42,7 +43,9 @@
     "test": "mocha --full-trace --check-leaks",
     "precover": "npm run lint && npm run beautify-lint",
     "cover": "istanbul cover node_modules/mocha/bin/_mocha",
-    "travis": "npm run cover -- --report lcovonly"
+    "travis": "npm run cover -- --report lcovonly",
+    "release:create": "standard-version -i CHANGELOG.md -m \"Creating release %s\"",
+    "release:publish": "git push --follow-tags origin master; npm publish"
```

Comment:
Also worth noting that we want to eventually automate these, the command execution needs to do a single task to allow finite script control without bloating the scripts with what would end up being duplicate functionality.